Config/1_novathesis.tex

```diff
@@ -26,6 +26,7 @@
 %                               %       iscteiul/eta, ips/ests, ipl/isel, ulht/deisi
 %                               %       other/esep, other/mscgt ]
 %                               % DEFAULT: school=nova/fct
+\ntsetup{copyrightmodifier=by}
```

Comment:
If the CC license is only for uminho, we must think "where" to add this customization option.  I'm not sure if this is the right location, as it may mislead the other users.  Probably the best option is to “take out” the CC license from uminho and make it available to everybody.  Not sure how to do it now…  probably I'll accept your pull request and then work over it…

Reply:
Can I mark this as resolved then?